packages/kbn-apm-synthtrace/tsconfig.json

```diff
@@ -8,7 +8,6 @@
   "kbn_references": [
     "@kbn/datemath",
     "@kbn/apm-synthtrace-client",
-    "@kbn/apm-plugin"
```

Comment:
i don't think this was intended, it was from an auto-commit by elasticmachine, will revert this line

Reply:
nevermind, this line is correct. not sure why it existed previously in the tsconfig.json, but it's not referenced in any of the package imports.